tests/saw/Makefile

```diff
@@ -86,14 +87,19 @@ tmp/%.log : %.saw bitcode/all_llvm.bc tmp
 ## BIKE Tests
 ###########################################
 
+BIKE_SCRIPTS=$(wildcard bike/*.saw)
+
 .PHONY : bike
 bike : bike_bitcode
-	@${MAKE} clean-bike-logs
-	@${MAKE} bike/bike.log
+	# @${MAKE} clean-bike-logs
+	# @${MAKE} bike/bike.log
+	# @${MAKE} bike/verify_bike.saw
+	@${MAKE} -C bike force_all
 
-bike/bike.log :
-	@set -o pipefail; \
-	saw bike/verify_bike.saw | tee $@
+# bike/%.saw: bike_bitcode bike/tmp/%.log
+
+bike/%.saw: bike_bitcode
```

Comment:
`bike/tmp/%.log`? Also, is this target used in this makefile?

Reply:
No, it has been removed.